pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/HadoopSegmentPreprocessingJob.java

```diff
@@ -60,9 +60,9 @@
 import org.apache.pinot.hadoop.job.partitioners.GenericPartitioner;
 import org.apache.pinot.hadoop.job.reducers.SegmentPreprocessingReducer;
 import org.apache.pinot.hadoop.utils.PinotHadoopJobPreparationHelper;
-import org.apache.pinot.ingestion.common.ControllerRestApi;
-import org.apache.pinot.ingestion.common.JobConfigConstants;
-import org.apache.pinot.ingestion.jobs.SegmentPreprocessingJob;
+import org.apache.pinot.plugin.ingestion.batch.common.ControllerRestApi;
```

Comment:
It might be better to not re-organize packages in v0_deprecated

Reply:
Ok, I have reverted the package renaming for all of pinot-batch-ingestion.